packages/dds/tree/README.md

```diff
@@ -320,3 +320,24 @@ flowchart
     end
     package-->runtime["Fluid runtime"]
 ```
+
+# Open Design Questions
```

Comment:
I think this might deserve its own page so the README can be a more atomic view of the current state of the tree.

Reply:
This readme tries to detail the actual goals of this package, and how its architected to meet those goals. The known design issues where its design (as covered in this document) doesn't work seem in scope.

Also I think the fact that significant parts of the codebase don't really work due to design issue is part of the current state of this package. Note that the design issues here all impact the architectural role of top level submodules in this package in a way that when fixes will likely require changes to the architectural details covered above.

I'm updating the description to clarify this.

```suggestion
# Open Design Questions

The design issues here all impact the architectural role of top-level modules in this package in a way that when fixed will likely require changes to the architectural details covered above.
Smaller scoped issues which will not impact the overall architecture should be documented in more localized locations.
```